### DISCUSSION TITLE
HESK/ UVDesk

### DISCUSSION THREAD

**Discussion by tech2mar** [tteck/Proxmox#2008] [Ideas]
Maybe we will get a HESK or UVDesk LXC or anything else like that ?

This would be great

**Comment by tteck** [comment]
Would https://www.faveohelpdesk.com/ work for you?
If so, it's in the TurnKey LXC Appliances script.

**Reply by tech2mar** [reply]
Yes for sure, thanks for the quick reply and the great work, ou are doing here.
